x-pack/metricbeat/metricbeat.reference.yml

```diff
@@ -131,6 +131,15 @@ metricbeat.modules:
   # Diskio configurations
   #diskio.include_devices: []
 
+#------------------------------- Activemq Module -------------------------------
+- module: activemq
+  metricsets: ['broker', 'queue', 'topic']
+  period: 10s
+  hosts: ['localhost:8161']
+  path: '/api/jolokia/?ignoreErrors=true&canonicalNaming=false'
+  username: admin # default username
+  password: admin # default password
```

Comment:
I understand these are the actual defaults? this should be commented out if not

Reply:
The default installation of ActiveMQ uses admin/admin credentials. I wouldn't like to modify the out-of-the-box setup.